### PR TITLE
fixed incoming transaction fiat amount

### PR DESCRIPTION
**Pull request by Anschutz1927**

### CODE REVIEW

File: app/src/main/java/io/multy/util/CryptoFormatUtils.java [hunk 1]
```diff
@@ -27,7 +27,14 @@ public static String satoshiToBtc(long satoshi) {
         return result;
     }
 
-    public static String satoshiToUsd(long satoshi) {
+    public static double satoshiToBtcDouble(long satoshi) {
```
Comment (AndreyGrowl) [inline app/src/main/java/io/multy/util/CryptoFormatUtils.java]: Heard about method overloading? Also test first. These changes affect too much places.

File: app/src/main/java/io/multy/util/CryptoFormatUtils.java [hunk 2]
```diff
@@ -27,7 +27,14 @@ public static String satoshiToBtc(long satoshi) {
...
+        if (satoshi == 0) {
+            return 0.0;
+        }
+        return satoshi / Math.pow(10, 8);
+    }
+
+    public static String satoshiToUsd(double satoshi) {
```
Comment (AndreyGrowl) [inline app/src/main/java/io/multy/util/CryptoFormatUtils.java]: Once again. Satoshi cany be ONLY long

File: app/src/main/java/io/multy/util/CryptoFormatUtils.java (hunk 1 above)
Comment (Anschutz1927) [reply app/src/main/java/io/multy/util/CryptoFormatUtils.java]: please, try to see full file https://github.com/Anschutz1927/Multy-android/blob/b8bae6c8ce9550efe12ec893340d50b4c3810acc/app/src/main/java/io/multy/util/CryptoFormatUtils.java

File: app/src/main/java/io/multy/util/CryptoFormatUtils.java (hunk 2 above)
Comment (Anschutz1927) [reply app/src/main/java/io/multy/util/CryptoFormatUtils.java]: ok

**Pull request merged**